### ISSUE TITLE
Question: Keyboard navigation in Scatter plot

### ISSUE THREAD

**Issue by pan-kot** [highcharts/highcharts#23097]
Hi Highcharts team,

I have a question about keyboard navigation in a scatter plot. If I understood the behaviour correctly, pressing Arrow Up/Down moves the focus to the point with higher/lower y. I am not sure though, what happens when the Arrow Left/Right keys are used - could you please clarify?

I tried the navigation in this demo: https://www.highcharts.com/demo/highcharts/scatter

When I am pressing Arrow Left/Right keys the navigation look random:

https://github.com/user-attachments/assets/6ce805b0-6836-4808-9176-2cb2c494594e

Are there customisation options for keyboard navigation available? Ideally, I want the navigation to move to the closest point in both directions, or something similar, but it should be predictable so that the user can get to they point they want.

Thank you in advance!
